src/components/theme/application/applicationProvider.component.tsx

```diff
@@ -94,6 +101,9 @@ interface State {
  * );
  * ```
  */
+
+export const ApplicationContext = createContext({ topInsetEnabled: false });
```

Comment:
The insets are related to the modal's functionality.
The application provider should not handle it.